app/src/main/java/org/squidfish/tuplenback/data/LocalStorageRepository.kt

```diff
@@ -0,0 +1,15 @@
+package org.squidfish.tuplenback.data
+
+import org.koin.core.annotation.Single
+import org.squidfish.tuplenback.data.room.AppDatabase
+import org.squidfish.tuplenback.models.GameStatsModel
+
+@Single
```

Comment:
Can we refrain from annotating classes like this? If we decide to swap out Koin, it would have much smaller impact if we need to update only the module files (and the Application's `startKoin` call